Optix Group 4 Code/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems;
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.*;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+/**
+ * @author Chilliam Weng
+ * @author Advik Garg
+ * @author Don Tran
+ * 
+ * This code is for the Intake subsystem, it defines the solenoids, neo motors, and compressor and contains all the necessary methods
+*/
+public class Intake extends SubsystemBase {
+    Compressor pcmCompressor = new Compressor(Constants.Intake.compressor, PneumaticsModuleType.CTREPCM);
```

Comment:
Motor objects should be encapsulated within the subsystem. Make the objects private.